test/PHPQueue/Backend/PredisZsetTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+namespace PHPQueue\Backend;
+class PredisZsetTest extends \PHPUnit_Framework_TestCase
+{
+    private $object;
+
+    public function setUp()
+    {
+        parent::setUp();
+        if (!class_exists('\Predis\Client')) {
+            $this->markTestSkipped('Predis not installed');
+        } else {
+            $options = array(
+                'servers' => array('host' => '127.0.0.1', 'port' => 6379)
```

Comment:
Should this go somewhere central? (It's repeated in a few places... Just a thought, may not be worth consolidating, tho...)